src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -48,7 +58,9 @@ class PaymentsPage extends React.Component {
      */
     paymentMethodPressed(nativeEvent, account) {
         if (account) {
-            // TODO: Show the make default/delete popover
```

Comment:
Are you planning to create separate issue for this TODO item?

Reply:
I'm not sure, but we can remove the TODO